docs/usage-guide/command-line.md

```diff
@@ -2,12 +2,12 @@
 
 In addition to providing an interactive terminal based UI to ask the user each question, `columbo` can also generate a
 command line argument parser based on the list of `Interaction`s. When used in this manner, `Echo` & `Acknowledge` are
-ignored. To produce a consistent command line argument format, the value of each question's `name` will be transformed
-using the following rule:
+ignored. To produce a consistent command line argument format, `columbo` will do the following to the value of each
+question's `name`:
 
-* If there are any upper-case characters, lower-case characters will be used.
-* If there are is a space character, a dash will be used in its place.
-* If there are is an underscore character, a dash will be used in its place.
+* Make it lower-cased
+* Replace space character with dash
+* Replace underscore character with dash
```

Comment:
I think _examples_ would help a lot on each of these lines. Maybe like so:

```markdown
* Make it lower-cased. E.g. "BIRTHdate" becomes "birthdate"
* Replace space characters with dashes. E.g. "is it raining" becomes "is-it-raining"
* Replace underscore characters with dashes. E.g. "is_it_sunny" becomes "is-it-sunny"
```

Reply:
Good suggestion. In the readme, there are some examples below these bullet points (see https://github.com/wayfair-incubator/columbo/blob/6a396571024f3995f2b7c266f6cf3deb534b3f1d/docs/usage-guide/command-line.md).

For now, I'm going to merge this PR, but we can certainly provide some in-line examples in another PR if you think that would be helpful.

Reply:
Thanks for pointing out 👍 I think as long as their are examples then all is good (inline or underneath is fine either way. 🙂 )